src/tapis-ui/components/systems/SystemList/SystemList.tsx

```diff
@@ -28,15 +28,16 @@ const SystemItem: React.FC<SystemItemProps> = ({ system, select, selected = fals
 
 interface SystemListProps {
   onSelect?: OnSelectCallback | null,
-  className?: string
+  className?: string,
+  params?: Systems.GetSystemRequest
 }
 
-const SystemList: React.FC<SystemListProps> = ({ onSelect=null, className=null }) => {
+const SystemList: React.FC<SystemListProps> = ({ onSelect=null, className=null, params=undefined }) => {
```

Comment:
Would it be easier to make `params` a required prop and conditionally render the system list only when they're defined?